signer/src/storage/model.rs

```diff
@@ -542,17 +542,18 @@ pub struct EncryptedDkgShares {
 /// Persisted public DKG shares from other signers
 #[derive(Debug, Clone, Hash, PartialEq, Eq, PartialOrd, Ord, sqlx::FromRow)]
 #[cfg_attr(feature = "testing", derive(fake::Dummy))]
-pub struct RotateKeysTransaction {
+pub struct KeyRotationEvent {
     /// Transaction ID.
     pub txid: StacksTxId,
+    /// The Stacks block ID of the block that includes the transaction
+    /// associated with this withdrawal request.
```

Comment:
nit:
```suggestion
    /// associated with this key rotation event.
```

Reply:
Updated in https://github.com/stacks-sbtc/sbtc/pull/1599/commits/3933b622c98731a777d6c7f38e6fbd88d556502d.